### DISCUSSION TITLE
Runway help

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#5473] [Q&A] [answered]
After i created model and config the runway resources and try to run the migration using : php please runway:generate-migrations resource-handle and replaced resource-handle with my model i got this error : Runway could not find [Order]. Please ensure its configured properly and you're using the correct handle. How to fix it please

**Comment by duncanmcclean** [comment, marked as answer]
Runway is a third-party addon, please open an issue on [Runway's repository](https://github.com/doublethreedigital/runway/issues).

**Reply by HazemMohamed252** [reply]
Thanks
